models/multimeter.h

```diff
@@ -138,9 +139,9 @@ namespace nest
  * with a given time stamp.
  *
  * Start and stop are handled as follows: the first recorded
- * data is with time stamp origin+start+1, the last recorded one
- * that with time stamp origin+stop. Only such times are recorded
- * for which (T-(origin+start)) mod interval is zero.
+ * data is with time stamp offset+start+1, the last recorded one
+ * that with time stamp offset+stop. Only such times are recorded
+ * for which (T-(offset+start)) mod interval is zero.
```

Comment:
start, stop and origin are independent of offset and interval. Isn't the correct description now "Data is recorded at time steps T for which start < T <= stop and ( T - offset ) mod interval == 0."?